### DISCUSSION TITLE
vo 3줄 요약

### DISCUSSION THREAD

**Discussion by seong-wooo** [woowa-Effective-Java-Study/effective-java-study#7] [General]
1. int, String 등 과 같이 값의 형태를 갖는 자료형을 새롭게 만든 것
2. 불변, 동등, 주고 받을 수 있음  3가지 성질이 있다.
3. 값에 규칙이 존재할 경우 vo로 생성하라

지극히 제 의견

**Comment by 2yujeong** [comment]
그래서 VO가 뭐죠?
